buildbot/buildbot/worker.py

```diff
@@ -214,20 +223,35 @@ def stage_make(db, config):
     generated.
     """
     prefix = config["HLS_COMMAND_PREFIX"]
-    with work(db, state.MAKE, state.MAKE_PROGRESS, state.HLS_FINISH) as task:
+    with work(db, state.MAKE, state.MAKE_PROGRESS, stage_after_make) as task:
         _task_config(task, config)
 
         if task['sdsflags']:
             task.log('WARNING: make stage is ignoring sdsflags={}'.format(
                 task['sdsflags']
             ))
-
-        make_cmd = prefix + [
+        # if toochain set to f1, use different make command.
+        if (TOOLCHAIN == 'f1'):
```

Comment:
Instead of importing the `TOOLCHAIN` variable, use `config['TOOLCHAIN']`. (Otherwise, you'll always get the default value, even if someone overrides it in their local config.)

No parentheses are necessary for `if`s in Python.